modules/DMZ/models/regression/rpart_classification.py

```diff
@@ -0,0 +1,93 @@
+"""
+Ryan Berg 10/19/16
+"""
+
+import sys
+import pandas as pd
+import numpy as np
+from rpy2.robjects import DataFrame
+from rpy2.robjects import pandas2ri
+from rpy2.robjects.packages import  importr
+pandas2ri.activate()
+
+# Imports for the R libraries needed
+rparty = importr("rpart")
+base = importr("base")
+stats = importr("stats")
+
+class Rpart:
+
+
+    def __init__(self):
+        self.model = pd.DataFrame
+        self.error_matrix = pd.DataFrame
+        self.outcome = ""
+        self.accuracy = 0
+
+
+    #Trains the model.
+    def fit(self, training_data,  target):
+        """
+        :param training_data: a pandas dataframe.
+        :param target: a string referring to the target varialbe to be predicted
+        :return: An rpart model
+        """
+
+        self.outcome = target
+
+        #Converting to proper format for R functions
+        train_data = DataFrame(training_data)
+        formula = target + " ~ ."
```

Comment:
Is the formula always going to be this? Do we always want all the non-results to be the predictors?